### DISCUSSION TITLE
How do I run this?

### DISCUSSION THREAD

**Discussion by jaydubbs3** [gwleuverink/clickup-time-tracker#21] [General]
Sorry I am a noob. I just downloaded the zip and want to run it.

What do I need to do to run the zip? 

Thanks in advance

**Comment by gwleuverink** [comment]
Hey, what OS are you using? 

You can download a executable from the releases page. You can find the files for each platform there.

Windows you can run the .exe, for Mac the .dmg & for Linux the .AppImage.

There is a caveat when running it on Mac, since the app is not signed. (I don't have a Apple dev account)

I believe you should install the app from the dmg file. Then press option while right clicking the app and click 'open'. After doing this once you can open the app like you normally would.

I will add this to the readme soon
